packages/terra-embedded-content-consumer/CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+* Added
+  * Added visually hidden text before and after the iframe to provide contexts to screen reader users on when the embedded content will begin and end.
```

Comment:
I don't think a consumer needs to know about the internal implementation, just the result. Let's update it to:
```suggestion
  * Added screenreader support to announce context of embedded iframe content.
```

Reply:
Updated here: https://github.com/cerner/terra-framework/pull/1804/commits/30f50f9daf5e5969fb40d1e52de1aac0bdb0d18c